enterprise_access/apps/subsidy_access_policy/tests/test_models.py

```diff
@@ -569,6 +569,24 @@ def test_content_would_exceed_limit_positive_spent_amount(self):
         with self.assertRaisesRegex(Exception, 'Expected a sum of transaction quantities <= 0'):
             self.per_learner_enroll_policy.content_would_exceed_limit(10, 100, 15)
 
+    def test_spend_limit_sum_and_content_price_equal_to_remaining_budget(self):
+        """
+        Ensures that passing a spent_amount equal to the remaining budget will return True.
+        """
+        self.assertFalse(self.per_learner_enroll_policy.content_would_exceed_limit(-90, 100, 10))
+
+    def test_spend_limit_sum_and_content_price_less_than_remaining_budget(self):
+        """
+        Ensures that passing a spent_amount equal to the remaining budget will return True.
+        """
+        self.assertFalse(self.per_learner_enroll_policy.content_would_exceed_limit(-90, 100, 5))
+
+    def test_spend_limit_sum_and_content_price_greater_than_remaining_budget(self):
+        """
+        Ensures that passing a spent_amount equal to the remaining budget will return True.
+        """
+        self.assertTrue(self.per_learner_enroll_policy.content_would_exceed_limit(-90, 100, 100))
```

Comment:
nit: I might set this up so that the content price is 11, so that going only 1 cent over the limit to check means that `content_would_exceed_limit` returns true.